src/Controls/src/Core/Element/Element.cs

```diff
@@ -280,6 +280,7 @@ internal bool RemoveLogicalChild(Element element, int index)
 
 		internal bool Owned { get; set; }
 
+		Element _strongParentOverrideReference;
```

Comment:
This is going to break other scenarios with memory leaks. The main point of the weakreference on the Parent is to avoid circular references on iOS which cause things to not get collected. So, we'll need to understand a bit more about why the Parent here is getting collected at all. 

The parent should ideally be rooted to something so that doesn't happen, but it looks like there's probably a scenario we're not accounting for